src/Commands/HydroponicsDrushCommands.php

```diff
@@ -92,22 +92,25 @@ public function hydroponics(
       }
       if ($all_idle === 1) {
         $loop->cancelTimer($timer_ping);
-          foreach($done as $queue_timer) {
-            $loop->cancelTimer($queue_timer);
-          }
-          \Drupal::state()->set('hydroponics.queurunner_last_pid', 0);
+        foreach($done as $queue_timer) {
+          $loop->cancelTimer($queue_timer);
+        }
+        \Drupal::state()->set('hydroponics.queurunner_last_pid', 0);
+        $loop->stop();
```

Comment:
Hi, oh sorry my fault. I meant instead of $loop->stop() $loop->cancelTimer($timer). Just its own cleanup...